src/components/Accordion/PAccordionContent.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <AccordionContent
+    v-bind="delegatedProps"
+    class="p-accordion-content"
+  >
+    <div class="pb-4 pt-0" :class="props.class">
+      <slot />
+    </div>
```

Comment:
Why an interior div instead of something like:
```vue
<template>
  <AccordionContent
    v-bind="delegatedProps"
    class="p-accordion-content"
  >
  <slot />
</AccordionContent>

...

<style>
.p-accordion-content { @apply
  pb-4
  pt-0
}
</style>
```

Generally using an arbitrary inner element like this makes the component less flexible than intended and more difficult to style, whether because you need to pass classes to the inner div like you're doing here OR you need to pass the to the outer one and can't.